### ISSUE TITLE
`"allowSameFolder": true` should still catch `./../`

### ISSUE THREAD

**Issue by SamuelT-Beslogic** [MelvinVermeer/eslint-plugin-no-relative-import-paths#30]
Config:
```json
"no-relative-import-paths/no-relative-import-paths": [
  "error",
  { "allowSameFolder": true }
]
```
Catches the following: `../path/to/relative/parent/import`
But not: `./../path/to/relative/parent/import`

**Comment by SamuelT-Beslogic** [comment]
`allowedDepth` option was recently added. That would have to be taken into account as well (allowed depths of 1 would allow `/../` but not `/../../`)
